README.md

```diff
@@ -145,17 +162,17 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
-
+create executable:  g++ test.cpp
+run executable:     ./a.out
```

Comment:
The intent of this exercise was to create an actual data structure containing the table and then use it to drive the interpreter. We discussed this on Wednesday.

You produced a manual parser that really has no connection to the state transitions in the previous two sections.

# -6